.prettierignore

```diff
@@ -0,0 +1,2 @@
+.env
+.env.example
```

Comment:
Does it try and format these if you don't add this?

Reply:
It did at first so I added them, but let me try to remove now and see what happens.